plugins/org.yakindu.sct.generator.java/src/org/yakindu/sct/generator/java/JavaExpressionsGenerator.xtend

```diff
@@ -61,10 +64,14 @@ class JavaExpressionsGenerator extends ExpressionsGenerator {
 	override dispatch String code(AssignmentExpression it) {
 		if (varRef.definition instanceof Property) {
 			var property = varRef.definition as Property
-			if (eContainer instanceof Expression) {
-				return '''«property.getContext»«property.assign»(«assignCmdArgument(property)»)'''
+			if (property.eContainer instanceof LocalVariableDefinition) {
+				return '''«property.getContext»«property.name» = «assignCmdArgument(property)»;'''
```

Comment:
I think you need to remove the semicolon

Reply:
has been removed